src/cnxml/de.ts

```diff
@@ -381,8 +384,8 @@ export function normalizeVoid(editor: DeserializingEditor, at: Path): void {
         normalizeWhiteSpace(editor, at)
     }
 
-    if (node.children.length === 1
-    && Text.isText(node.children[0]) && node.children[0].text === '') {
+    if (node.children.length > 0
+    && node.children.every((child: Slate.Node) => Text.isText(child) && child.text === '')) {
```

Comment:
I think this could stay the same if we update line `600` from this diff to `children: [],` instead of `children: [{ text: '' }],` but I feel like checking all children instead of the only first one is just better

Reply:
This check is right after a call to `normalizeWhiteSpace`, so any sequence of `Text` nodes should already have been collapsed into a single `Text` node. This means that at this point the void _must_ have exactly one empty `Text` child. If it doesn't then it will be normalized, and deserialization will fail at line 146.